### PR TITLE
Publish @brightlayer-ui/react-native-components v7.0.0

### PR DESCRIPTION
**Pull request by daileytj**
Fixes # .

#### Changes proposed in this Pull Request:
- Publish @brightlayer-ui/react-native-components v7.0.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+## v7.0.0 (October 14, 2022)
+
+### Changed
+
+-   Update to build with react-native 0.70.0.
+-   Update peer dependencies to use "react-native-vector-icons": "^9.0.0".
```
Comment (joebochill) [inline CHANGELOG.md:8]: match the suggested wording from the icons PR

File: components/src/core/drawer/utilities.tsx [hunk 2]
```diff
@@ -11,8 +11,11 @@ import { AllSharedProps } from './types';
  * @param childValue The value of the property on the child
  * @returns The child value if it is defined, otherwise the parent value
  */
+
+/* eslint-disable */
 export const mergeStyleProp = <T extends unknown>(parentValue: T, childValue: T): T | undefined =>
```
Comment (joebochill) [inline components/src/core/drawer/utilities.tsx:16]: Why is eslint complaining about this function?

Comment (daileytj) [reply components/src/core/drawer/utilities.tsx:16]: welp... I was wrong. 

```
15:32  error  Constraining the generic type 'T' to 'unknown' does nothing and is unnecessary  @typescript-eslint/no-unnecessary-type-constraint

✖ 1 problem (1 error, 0 warnings)

error Command failed with exit code 1.
```

Comment (joebochill) [reply components/src/core/drawer/utilities.tsx:16]: If you wanted to, you could just take off the `extends unknown` part. But either way is fine.

**Review by joebochill — APPROVED** [review]

**Review by huayunh — APPROVED** [review]
Slapping an approve to trigger a publish

**Pull request merged**